evaluation/benchmarks/gorilla/run_infer.py

```diff
@@ -55,6 +55,8 @@ def get_config(
         workspace_mount_path=None,
     )
     config.set_llm_config(metadata.llm_config)
+    agent_config = config.get_agent_config(metadata.agent_class)
```

Comment:
there's probably a better way to DRY this out, but I'm not familiar enough with eval. E.g. maybe a shared config.toml? I dunno

Reply:
I think it was intentional to explicitly write these out - so eval is not gonna be mysteriously impacted by a local `config.toml` (which did happen in the early days and took me a while to figure out 😢 )